metrix-tools/src/test/java/com/powsybl/metrix/tools/MappingToolTest.java

```diff
@@ -101,7 +101,7 @@ void runMappingIsIncomplete() throws IOException {
             "--mapping-file", "/mapping.groovy",
             "--time-series", "/timeseries.csv"
         };
-        assertCommand(commandLine, CommandLineTools.COMMAND_OK_STATUS, expected.toString(), "Mapping is incomplete\n");
+        assertCommand(commandLine, CommandLineTools.COMMAND_OK_STATUS, expected.toString(), "Mapping is incomplete");
```

Comment:
ℹ️ this was the windows issue

Reply:
Maybe it might be interesting to add the windows tests in the GitHub CI for the Java part?

Reply:
indeed, added maven wrapper & windows and macos build to make it consistent with other powsybl repos

Reply:
thanks!